### PR TITLE
Update jonomai, shōdan spelling, popup text

### PR DESCRIPTION
**Pull request by broadwell**
Addresses #423, other requests by PIs:
- In-code references to jo-no-mai changed to jonomai so that catalog link now works; test fixtures and snapshots updated automatically
- Popup text for link to catalog of shōdan in Level 2 now just reads "Definition"
- Update spelling of shōdan in readme

### CODE REVIEW

**Comment by simonwiles** [comment]
I have a couple of questions about this one:  first, I'd like to clean up `src/form.html`.  It looks like a code editor or similar on your system has done some automatic conversion of Mac line-endings and mangled it a bit, and I can't see what's changed very well -- it looks like a `jo-no-mai` has changed to `jonomai` in the first commit, and then back again in the third commit, but there also seem to be some changes to `<br/>` tags and `<td>` tags that are hard to follow.

More materially, though, I want to understand what's going on with these alternative spellings.  You wrote to Jarek
> An important point to note, however, is that because of how the data processing pipeline currently operates, it is necessary that the URL portion of any link to the jonomai *catalog* entry should still be spelled jo-no-mai...

This feels very wrong -- is there a reason we can't just fix this?  Would you mind if we took a look together?

**Comment by broadwell** [comment]
Sorry about `form.html`, whatever happened there. But as to the renaming question, yeah, we should revisit this. The PR ended up being a bit of a moving target, so it's better to start from scratch. I believe the root cause is that there are basically two different slugifying regimes, based on different fields in the data spreadsheets, for the catalog links and L2 shodan links. Fortunately (I think) it only seems to be a problem for this section and maybe aikyogen/ai-kyogen.
